tests/test_qpu.py

```diff
@@ -2,18 +2,21 @@
 Unit tests for the QPU device.
 """
 import logging
+import re
 
 import pytest
 import pyquil
 import pennylane as qml
 from pennylane import numpy as np
-from conftest import BaseTest
+import pennylane_forest as plf
+from conftest import BaseTest, QVM_SHOTS
 
+from flaky import flaky
 
 log = logging.getLogger(__name__)
 
-VALID_QPU_LATTICES = [qc for qc in pyquil.list_quantum_computers() if "qvm" not in qc]
-
+pattern = 'Aspen-.-[1-5]Q-.'
+VALID_QPU_LATTICES = [qc for qc in pyquil.list_quantum_computers() if "qvm" not in qc and re.match(pattern, qc)]

```

Comment:
Good catch. For context: previously, VALID_QPU_LATTICES would return reasonably small sized lattices for the test suite to run on. However, with Aspen-7 now available, a lot of these lattices were significantly larger, causing the test suite to hang/run out of memory 🙂